File: fjt.pricingservice/MySqlDBModel/rfq_assy_autopricingstatus.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace fjt.pricingservice.MySqlDBModel
{
    [Table("rfq_assy_autopricingstatus")]
    public partial class rfq_assy_autopricingstatus
    {
        [Key]
        public int id { get; set; }
        public int rfqAssyID { get; set; }
        public int? status { get; set; }
        public string pricingApiName { get; set; }
        public string msg { get; set; }
        public string errorMsg { get; set; }
        public string userID { get; set; }
        public DateTime statusChangeDate { get; set; }
        public int? pricingSupplierID { get; set; }
    }
}

File: fjt.api/models/RFQAssyAutoPricingStatus.js
const Sequelize = require('sequelize');

module.exports = (sequelize) => {
    const RFQAssyAutoPricingStatus = sequelize.define('RFQAssyAutoPricingStatus', {
        id: {
            allowNull: false,
            autoIncrement: true,
            primaryKey: true,
            type: Sequelize.INTEGER
        },
        rfqAssyID: {
            allowNull: false,
            type: Sequelize.INTEGER
        },
        pricingApiName: {
            type: Sequelize.STRING,
            allowNull: true
        },
        status: {
            type: Sequelize.INTEGER,
            allowNull: true

        },
        msg: {
            type: Sequelize.TEXT,
            allowNull: true
        },
        errorMsg: {
            type: Sequelize.TEXT,
            allowNull: true
        },
        userID: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        statusChangeDate: {
            type: Sequelize.DATE,
            allowNull: false
        },
        isPurchaseApi: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        pricingSupplierID: {
            type: Sequelize.INTEGER,
            allowNull: true
        }
    }, {
        paranoid: true,
        timestamps: false,
        tableName: 'rfq_assy_autopricingstatus'
    });
    return RFQAssyAutoPricingStatus;
};
